packages/components/src/components/Alert/Alert.tsx

```diff
@@ -194,6 +180,43 @@ export const Alert: FC<AlertProps> = ({
     lineHeight: 30,
   }
 
+  const _header = () => {
+    const headerText = header ? <Text style={headerFont}>{header}</Text> : null
+    const a11yLabel = header ? headerA11yLabel || header : ''
+
+    /**
+     * Wrap header text and expand icon in Pressable if the Alert is expandable
+     * Otherwise wrap in View with accessibility props
+     */
+    if (expandable) {
+      return (
+        <Pressable
+          onPress={() => setExpanded(!expanded)}
+          accessibilityRole="tab"
```

Comment:
1. Swap to `role` for the new/more-web-like accessibility props
2. Is this the right role? I don't see anything better looking at the list, but doesn't really seem like a tab per se... maybe should confirm with Jessica/Brea?

Reply:
This was done based on CollapsibleAlert. Looks like Brea had already checked off on `tab` being the correct role. Although I'm now seeing that it's just the header that should be a `tab` and the entire View should be a `tablist`. Corresponding ticket here: https://github.com/department-of-veterans-affairs/va-mobile-app/issues/4365#issuecomment-1363362436